src/docs/product/accounts/quotas/manage-transaction-quota.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Manage Your Transaction Quota
+keywords: ["best practices"]
+sidebar_order: 20
+description: "Learn how to use the tools Sentry provides to control the type and amount of transactions that are monitored."
+---
+
+In most cases, sending all transaction events to Sentry would generate more performance data than you might find useful and might use up your events quota too quickly. Sentry provides tools to control the _type_ and _amount_ of transactions that are monitored. This allows you to have transaction data that's actionable and meaningful, and to manage your quota if you're on an event-quota based plan. Applying the proper SDK configuration is an iterative and on-going process, but these tips will show you how to best use Sentry's tools to get the most out of your transaction events.
```

Comment:
```suggestion
In most cases, sending all transaction events to Sentry would generate more performance data than you might find useful and might use up your transactions quota too quickly. Sentry provides tools to control the _type_ and _amount_ of transactions that are monitored. This allows you to have transaction data that's actionable and meaningful, and to manage your quota. Applying the proper SDK configuration is an iterative and on-going process, but these tips will show you how to best use Sentry's tools to get the most out of your transaction events.
```

Should we be more explict on which quota is consumed?  I don't know if we need to mention 'if you are on an event-quota based plan' as all transaction customers will be on one of those. 🤷

Reply:
both of these suggestions make sense to me. I'll apply them to the errors quota guide as well.